cli/tests/test_parallelize_command.rs

```diff
@@ -505,12 +551,21 @@ fn test_parallelize_failure_multiple_roots_with_different_parents() {
     ◉  000000000000
     "###);
 
-    insta::assert_snapshot!(
-    test_env.jj_cmd_failure(
+    test_env.jj_cmd_ok(
```

Comment:
nit: remove "failure" from the test name.

Reply:
Done